ang/civicase/Utils.js

```diff
@@ -154,18 +163,50 @@
       });
       return item;
     };
+
+    /**
+     * To count overdue tasks.
+     *
+     * @param {Array} activities - Array of related activities to a case
+     */
+    function countOverdueTasks (item) {
```

Comment:
why not use activities instead of item for the name?

Reply:
It was wrong. It should be the case, so using `caseObj`.